docs/tutorial/modules.rst

```diff
@@ -151,7 +186,10 @@ directive, for example:
     toTree (x :: xs) = insert x (toTree xs)
 
 In this case, any function with no access modifier will be exported as
-``abstract``, rather than left ``private``.
+``export``, rather than left ``private``.
```

Comment:
Is `private` the default now, or was the thing with switching on the presence of *any* access modifier left in?